### PR TITLE
Release 5.13.0

### PR DESCRIPTION
**Pull request by DomPeliniAerospike**

### CODE REVIEW

File: scripts/build-c-client.sh-cclient-output.log
Comment (realmgic) [inline scripts/build-c-client.sh-cclient-output.log:1]: should be removed or ignored

**Review by realmgic — DISMISSED** [review]

File: k.js
[the remark this answers is not in the document]
Comment (DomPeliniAerospike) [reply k.js:1]: Removed in latest commit.

File: scripts/build-c-client.sh-cclient-output.log
Comment (DomPeliniAerospike) [reply scripts/build-c-client.sh-cclient-output.log:1]: added this to .gitignore and removed the file.

File: ts-test/tests/apply.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
+// *****************************************************************************
+// Copyright 2013-2023 Aerospike, Inc.
```
[the remark this answers is not in the document]
Comment (DomPeliniAerospike) [reply ts-test/tests/apply.ts:2]: Updated all copyright strings.

File: ts-test/tests/aerospike.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+// *****************************************************************************
+// Copyright 2013-2023 Aerospike, Inc.
```
[the remark this answers is not in the document]
Comment (DomPeliniAerospike) [reply ts-test/tests/aerospike.ts:2]: Updated all copyright strings in latest commits.

File: lib/aerospike.js [hunk 3]
```diff
@@ -402,7 +416,7 @@ exports.log = as.log
  *
  * @property {number} NAMESPACE_DEFAULT - Use the default TTL value for the
  * namespace of the record.
- * @property {number} NEVER_EXIRE - Never expire the record.
+ * @property {number} NEVER_EXpIRE - Never expire the record.
```
[the remark this answers is not in the document]
Comment (DomPeliniAerospike) [reply lib/aerospike.js:419]: typo, fixed in latest commit

File: scripts/build-c-client.sh-cclient-output.log
[the remark this answers is not in the document]
Comment (DomPeliniAerospike) [reply scripts/build-c-client.sh-cclient-output.log:1]: Done

**Pull request merged**